web_src/js/svg.js

```diff
@@ -66,14 +75,25 @@ export const svgs = {
   'octicon-triangle-down': octiconTriangleDown,
   'octicon-x': octiconX,
   'octicon-x-circle-fill': octiconXCircleFill,
+  'octicon-plus': octiconPlus,
+  'octicon-search': octiconSearch,
+  'octicon-filter': octiconFilter,
+  'octicon-archive': octiconArchive,
+  'octicon-star': octiconStar,
+  'gitea-double-chevron-left': giteaDoubleChevronLeft,
+  'gitea-double-chevron-right': giteaDoubleChevronRight,
+  'octicon-chevron-left': octiconChevronLeft,
+  'octicon-organization': octiconOrganization,
 };
 
+// TODO: use a general approach to manage SVG icons
+
 const parser = new DOMParser();
 const serializer = new XMLSerializer();
 
 // retrieve a HTML string for given SVG icon name, size and additional classes
 export function svg(name, size = 16, className = '') {
-  if (!(name in svgs)) return '';
+  if (!(name in svgs)) return `<span>SVG:${name}</span>`;
```

Comment:
Better use a placeholder icon like a new `gitea-question` which would show a question mark:

```svg
<svg width="36" height="36" viewBox="-2 -2 36 36"><path d="M22.54 3.57c-1.82-1.31-4.11-1.97-6.87-1.97a9.6 9.6 0 0 0-5.3 1.39c-2.3 1.45-3.5 3.91-3.65 7.38H12c0-1.01.3-1.98.89-2.92.59-.94 1.59-1.4 3-1.4 1.44 0 2.43.37 2.97 1.13.54.77.8 1.6.8 2.53 0 .81-.4 1.55-.88 2.22A4.6 4.6 0 0 1 17.73 13s-2.87 1.84-4.13 3.31c-.73.86-.8 2.14-.86 3.98 0 .14.05.4.5.4h4.12c.4 0 .5-.3.5-.43.03-.67.1-1.01.22-1.4.23-.73.87-1.37 1.57-1.92l1.46-1c1.31-1.03 2.36-1.87 2.82-2.53a6.7 6.7 0 0 0 1.35-4c0-2.58-.91-4.53-2.74-5.85zm-6.95 20.3a3.18 3.18 0 0 0-3.39 3.2c-.05 1.97 1.37 3.28 3.2 3.33 1.9.05 3.36-1.17 3.42-3.14.06-1.98-1.33-3.33-3.23-3.38z"/></svg>
```

Reply:
Still, you should not use text where a SVG icon is expected, that will break layout.

Reply:
That's quite good to warn developers that they made a mistake.

Just like I added many errors (let me see)

Reply:
Hmm, maybe. Check backend code as well, I think it needs the same update too.

Reply:
The example:

* https://github.com/go-gitea/gitea/pull/18971#issuecomment-1158973862

Catch errors as early as possible, do not tolerate low-level mistakes.

Reply:
Still, I would not punish the user with this layout break in case a developer makes such a mistake. It's possible they may not catch the issue during their testing.

Reply:
Then why there were so many frontend bugs in Gitea .... I have fixed a lot, till recently, there are still many, like https://github.com/go-gitea/gitea/issues/22702

I wouldn't say it's "punish", instead, resolving bugs ASAP helps users. If we can't get error reports from users, we would never resolve the bugs.

Reply:
Maybe we should just throw an error instead of rendering? That would be more visible.

Reply:
No problem, done.